jujugui/static/gui/src/app/components/inspector-relations/inspector-relations.js

```diff
@@ -28,9 +28,16 @@ YUI.add('inspector-relations', function() {
     _generateRelations: function() {
       var relations = this.props.serviceRelations;
       var components = [];
-      relations.forEach(function(relation) {
+      if (relations.length === 0) {
+        return (
+          <div className="inspector-relations__message">
+            No relations for this service.
```

Comment:
I didn't want to add that yet as there is no where to click to add one. Will remember to update this message in the add-relation branch.